EngineeringModel/ViewModels/ElementDefinitionBrowser/ElementDefinitionsBrowserViewModel.cs

```diff
@@ -434,13 +444,19 @@ protected override void InitializeCommands()
 
             this.CreateSubscriptionCommand = ReactiveCommand.Create(this.WhenAnyValue(x => x.CanCreateSubscription));
             this.CreateSubscriptionCommand.Subscribe(_ => this.ExecuteCreateSubscriptionCommand());
-            
+
             this.BatchCreateSubscriptionCommand = ReactiveCommand.Create(this.WhenAnyValue(x => x.CanCreateBatchSubscriptions));
             this.BatchCreateSubscriptionCommand.Subscribe(_ => this.ExecuteBatchCreateSubscriptionCommand());
 
             this.ChangeOwnershipCommand = ReactiveCommand.Create();
             this.ChangeOwnershipCommand.Subscribe(_ => this.ExecuteChangeOwnershipCommand());
 
+            this.SetAsTopElementDefinitionCommand = ReactiveCommand.Create();
```

Comment:
I propose we add a canExecute statement for enabling/disabling the menu item. Have a look at the `this.WhenAnyValue(x => x...)` syntax

Reply:
Might be ` ReactiveCommand.CreateAsyncTask`

Reply:
Yes, but the item will be visible only on element definitions, and it will either be "Set as Top Element" for all elements except the top one, or "Unset as Top Element" for the top one, so I don't think we need it disabled at any state, correct?